### PR TITLE
improved homepage.json file

### PR DESCRIPTION
**Pull request by urvishp80**
- dashes instead of numbers and new lines in summary
- 12 active posts and 12 recent posts

### CODE REVIEW

**Review by Emmanuel-Develops — APPROVED** [review]

File: static/homepage.json [hunk 1]
```diff
@@ -1,52 +1,217 @@
 {
-    "header_summary": "The context discusses the v=2 nonces signing protocol of musig2 and the challenges of preventing attacks and maintaining confidentiality. It also addresses the concerns about the growth of \"inscriptions\" on the blockchain and the need for options to reject them. Additionally, it explores proposals for using MuSig2 signing for mutual closes and the complexities of pricing fees and implementing an ideal payment system.",
-    "enteries": [
+    "header_summary": "The context discusses the problems caused by spam attacks in the Bitcoin network, the potential risks of accepting unconfirmed inputs in coinjoin transactions, the issue of front-running in peer-to-peer marketplaces, concerns about privacy and decentralization in the Lightning Network, and the introduction of a new protocol for closing channels. The discussions highlight the need for action against spam attacks, the potential dangers of allowing unconfirmed inputs, suggestions for addressing front-running, concerns about privacy risks in the Lightning Network, and the proposed protocol for simplifying channel closures.",
```
Comment (adamjonas) [inline static/homepage.json]: `The context discusses` is a pretty bad opener for a hook on a website

File: static/homepage.json [hunk 2]
```diff
@@ -1,52 +1,217 @@
...
+        {
+            "id": "004014",
+            "title": "LN Summit 2023 Notes",
+            "link": "https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004014.html",
+            "authors": [
                 "Carla Kirk-Cohen"
-            ]
+            ],
+            "published_at": "2023-07-19T13:56:11+00:00",
+            "summary": "- In late June, the annual specification meeting took place in New York City.-The attendees attempted to take transcript-style notes, which can be found at this link.-The notes aim to capture the discussions that occurred during the summit, rather than providing a concise summary.",
```
Comment (adamjonas) [inline static/homepage.json]: that stubborn spacing

File: static/homepage.json [hunk 3]
```diff
@@ -1,52 +1,217 @@
...
-                "leohaf at orangepill.ovh"
+                "Antoine Riard"
             ],
-            "published_at": "2023-07-25T14:11:49+00:00",
-            "summary": "- The bug pertaining to the \"inscriptions\" developed by @rodarmor has caused a rapid growth in size on the blockchain, impacting the UTXO set and resulting in daily growth of 25MB.- Current options in Bitcoin Core do not effectively address the issue of inscriptions, with the only available patch being impractical for most users.- Implementing a feature that allows users to reject inscriptions in the mempool would help mitigate the negative impact and protect the integrity of the network.",
-            "n_threads": 1,
+            "published_at": "2023-07-18T20:18:49+00:00",
+            "summary": "- Last year, a new community process was introduced to specify covenants due to failures and discussions about proposals. -During this period, initiatives like the Bitcoin Contracting Primitives Working Group were established, holding monthly meetings and in-depth discussions. -Though the author will focus on Lightning development, they encourage others to continue efforts on covenant changes proposals.",
```
Comment (adamjonas) [inline static/homepage.json]: spacing

File: static/homepage.json [hunk 4]
```diff
@@ -1,52 +1,217 @@
...
+            "link": "https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-July/021792.html",
             "authors": [
                 "Tom Trevethan"
             ],
-            "published_at": "2023-07-25T16:05:48+00:00",
-            "summary": "- The v=2 nonces signing protocol of musig2 is designed to prevent the Wagner attack by blinding the challenge value c from the server.- The key aggregation coefficient of 1 for each key in statecoin transfer prevents \"rogue key attacks\" and ensures a valid signature over the aggregate key.- The updated key generation protocol involves the server generating a private key x1 and a public key X1, while the user generates a private key x2, a public key X2, and a random blinding nonce z.",
-            "n_threads": 13,
+            "published_at": "2023-07-24T07:46:11+00:00",
+            "summary": "- The implementation discusses the use of 2-of-2 Schnorr Musig2 for statechains, ensuring party 1 remains 'blinded'.-The security of blinded statechains relies on trusting the server's report on partial signatures.-The 2-of-2 musig2 protocol involves generating private-public key pairs, aggregating keys, and signing messages.",
```
Comment (adamjonas) [inline static/homepage.json]: spacing

**Pull request closed without merging**